setup.py

```diff
@@ -17,7 +17,10 @@ def get_version_from_pyver():
             raise ImportError('You must install pyver to create a package')
         else:
             return 'noversion'
-    version, version_info = pyver.get_version(pkg="datalake", public=True)
+    try:
+        version, version_info = pyver.get_version(pkg="datalake", public=True)
+    except IndexError:
+        return 'noversion'
```

Comment:
I had to add this because an install failed halfway through (I hadn't installed python-dev) and pyver was throwing an IndexError. It's probably not required for the common case.